index.html

```diff
@@ -59,245 +68,282 @@ <h1 class="title">I'm Jonas <br>Glad to see you</h1>
+            <div class="about-me-top">
+              <h1 class="about-me-title">About <br>Myself</h1>
+              <p class="about-me-description bio">I’m a software developer! I can help you build a product , 
+                  feature or website Look through some of my work and 
+                  experience! If you like what you see and have a project 
+                  you need coded, don’t hestiate to contact me.</p>
+              <p class="social-title">LET'S CONNECT</p>
+              <div class="about-social-media">
+                  <ul>
                       <li>
-                        <img src="images/javascript.png" alt="javascript icon">
-                        <span>Javascript</span>
+                          <a href="https://twitter.com/jonaswullah" target="_self">
+                              <img src="images/twitter.svg" alt="twitter handle">
+                          </a>
+                      </li>
                       <li>
-                        <img src="images/html.png" alt="html icon">
-                        <span>HTML</span>
+                          <a href="https://linkedin.com/in/jonas-wullah" target="_self">
+                              <img src="images/linkedin.svg" alt="linkedin handle">
+                          </a>
                       </li>
                       <li>
-                        <img src="images/css.png" alt="css icon">
-                        <span>CSS</span>
+                          <a href="#" target="_self">
+                              <img src="images/medium.svg" alt="medium handle">
+                          </a>
                       </li>
-                    </ul>
-                </li>
-
-                <li>
-                  <div class="areas-category">
-                    <p>Frameworks</p>
-                    <img src="images/right-arrow.png" alt="right arrow">
-                  </div>
-                  
-                </li>
-
-                <img src="images/separator-bottom.png" alt="separator line">
-
-                <li>
-                  <div class="areas-category">
-                    <p>Skills</p>
-                    <img src="images/right-arrow.png" alt="right arrow">
-                  </div>
-                  
-                </li>
-
-                <img src="images/separator-bottom.png" alt="separator line">
-
-              </ul>
+                      <li>
+                        <a href="#" target="_self">
+                            <img src="images/social4.svg" alt="social handle">
+                        </a>
+                    </li>
+                      <li>
+                          <a href="#" target="_self">
+                              <img src="images/angellist.svg" alt="angellist handle">
+                          </a>
+                      </li>
+                  </ul>
+              </div>
+  
+              <button class="resume-btn">Get my resume</button>
```

Comment:
- [ ] Kindly add button interaction styles to this button (hover, active...) as shown in Figma 👍 

![image](https://user-images.githubusercontent.com/50721479/207724313-de0905fd-4cd5-468e-bdce-76c593b5f2a9.png)

![image](https://user-images.githubusercontent.com/50721479/207724359-176eee0d-f0cb-4655-8e15-4dee21545b56.png)